components/dropdown/dropdown-opener-mixin.js

```diff
@@ -100,7 +112,8 @@ export const DropdownOpenerMixin = superclass => class extends superclass {
 	}
 
 	__getContentElement() {
-		return this.shadowRoot.querySelector('slot').assignedNodes().filter(node => node.hasAttribute && node.hasAttribute('dropdown-content'))[0];
+		return this.getContentSlot().assignedNodes()
+			.filter(node => node.hasAttribute && node.hasAttribute('dropdown-content'))[0];
```

Comment:
Sorry to suggest yet another alternate, but could this code not just iterate over ALL the slots, looking for the `dropdown-content` attribute? Like use `querySelectorAll('slot')`.

Reply:
Yep it could, I did consider that option as well, but wasn't super keen on repeatedly calling `assignedNodes` looking for the content when the subclass knows best how to provide it.  I don't feel strongly either way though, since it will rarely get used in either case.

Reply:
Yeah like in the current cases where there's only 1 slot it'll behave exactly the same and only where there's multiple slots will it do a bit more work. Was just thinking it'd be simpler for folks to implement and less things to document, but good either way.

Alternately, can it assume that it's always the main slot? So you could look for `slot:not([name])` to find the one without a name. 🤷

Reply:
Yeah I did consider that option briefly too.  Initially I was apprehensive about the potential for breaking a current consumer, however it looks like the only usages of `DropdownOpenerMixin` are the core dropdown openers, and they all used unnamed slots so I _think_ it would be a safe change. 
 It would require that any `DropdownOpenerMixin` consumer reserve the unnamed slot for the dropdown content, which is probably an acceptable constraint.  I think I'd be more inclined going this route (even though it limits the consumer) than iterating all the slots.  🤷‍♂️

Reply:
Ok, I went the `slot:not([name])` route.